recipes/dbcppp/all/patches/0001-dep-cmake-rem-kcd.patch

```diff
@@ -0,0 +1,316 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index d905b29..ea94456 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -5,53 +5,17 @@ project("libdbcppp" VERSION 0.1.0)
+ 
+ option(build_tests "Build tests" ON)
+ option(build_examples "Build examples" ON)
++option(build_tools "Build examples" ON)
+ 
+-set(CMAKE_CXX_STANDARD 17)
+-set(CMAKE_STATIC_LIBRARY_PREFIX "")
+-set(CMAKE_SHARED_LIBRARY_PREFIX "")
+-set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
++set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
+ 
+ if (CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
+ 	add_definitions("/bigobj")
+ endif()
+ 
+ include_directories("include")
+-include_directories("third-party/libxml2/include")
+-include_directories("third-party/libxmlmm/libxmlmm")
+ include_directories("third-party/boost")
```

Comment:
It's not using boost from conan?

Reply:
None of the public header files use boost, so right now I am using the boost that's included in their repo. Should I change this to one provided by conan? It's a pretty heavy dependency.

Reply:
The idea is that all the dependencies are managed by Conan - this allows the client to report conflicts and avoid linker problems mixing versions

Reply:
Fixed!